services/core-api/app/api/services/document_manager_service.py

```diff
@@ -33,6 +35,22 @@ def initializeFileUploadWithDocumentManager(cls, request, mine, document_categor
 
         return Response(str(resp.content), resp.status_code, resp.raw.headers.items())
 
+    @classmethod
+    def pushFileToDocumentManager(cls, file_content, filename, mine, document_category,
+                                  authorization_header):
+        folder, pretty_folder = cls._parse_upload_folders(mine, document_category)
+        data = {
+            'folder': folder,
+            'pretty_folder': pretty_folder,
+            'filename': filename,
+            'authorization': authorization_header
+        }
+        my_client = client.TusClient(cls.document_manager_url, headers=data)
```

Comment:
This is where you are unable to send parameters in the body of the request?

Reply:
Correct.